packages/slate-commons/src/commands/insertNodes.ts

```diff
@@ -9,6 +9,7 @@ import { isBlock } from './isBlock';
 import { isCursorInEmptyParagraph } from './isCursorInEmptyParagraph';
 import { isInline } from './isInline';
 import { isVoid } from './isVoid';
+import { roughlyNormalizeNodes } from './roughly-normalize';
 
 interface Options {
```

Comment:
_Unrelated question regarding this fix:_

I was curious about the absence of a `pnpm-lock.yaml` file in the root of this repository. I understand that there may be valid reasons for this decision, and I'm simply interested in understanding the rationale behind it.

Reply:
https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514